### PR TITLE
Print runtime information for debugging.

### PR DESCRIPTION
**Pull request by wujingyue**

### CODE REVIEW

**Comment by wujingyue** [comment]
!build

File: csrc/scheduler/normalization_utils.cpp [hunk 1]
```diff
@@ -1173,10 +1173,14 @@ bool compileTimeCheck(Fusion* fusion, ScheduleHeuristic schedule_heuristic) {
...
       reduction_scheduler_utils::getReductionType(reduction_tvs);
-  if (getPersistentHeuristicFor(reduction_type) != schedule_heuristic) {
+  const ScheduleHeuristic persistent_heuristic =
+      getPersistentHeuristicFor(reduction_type);
+  if (persistent_heuristic != schedule_heuristic) {
     scheduler_debug_utils::canScheduleRejectReason(
         schedule_heuristic,
-        "schedule_heuristic doesn't match with reduction type.");
+        "schedule_heuristic doesn't match with reduction type `",
+        persistent_heuristic,
```
Comment (liqiangxl) [inline csrc/scheduler/normalization_utils.cpp:1182]:
```suggestion
        persistent_heuristic,
```
```suggestion
        reduction_type,
```

File: csrc/scheduler/normalization_inner_outer.cpp [hunk 2]
```diff
@@ -48,10 +48,14 @@ bool InnerOuterPersistentKernelScheduler::canScheduleCompileTime(
...
       reduction_scheduler_utils::getReductionType(reduction_tvs);
-  if (normalization_scheduler_utils::getPersistentHeuristicFor(
-          reduction_type) != heuristicType()) {
+  const ScheduleHeuristic persistent_heuristic =
+      normalization_scheduler_utils::getPersistentHeuristicFor(reduction_type);
+  if (persistent_heuristic != heuristicType()) {
     scheduler_debug_utils::canScheduleRejectReason(
-        heuristicType(), "heuristicType() doesn't match with reduction type.");
+        heuristicType(),
+        "heuristicType() doesn't match with reduction type `",
```
Comment (liqiangxl) [inline csrc/scheduler/normalization_inner_outer.cpp:56]:
```suggestion
        "schedule_heuristic doesn't match with reduction type `",
```

File: csrc/scheduler/normalization_inner_outer.cpp [hunk 3]
```diff
@@ -48,10 +48,14 @@ bool InnerOuterPersistentKernelScheduler::canScheduleCompileTime(
...
+        persistent_heuristic,
```
Comment (liqiangxl) [inline csrc/scheduler/normalization_inner_outer.cpp:57]:
```suggestion
        reduction_type,
```

**Review by liqiangxl — APPROVED** [review]
Thanks for the fix.

**Pull request merged**